modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -83,20 +113,20 @@ spec:
       containers:
       - command:
         - /usr/local/bin/kube-scheduler
-        - --address=0.0.0.0
-        - --leader-elect=false
         - --scheduler-name=custom-scheduler <2>
```

Comment:
Is missing the --config option: https://github.com/openshift/openshift-docs/pull/36425/files#diff-bb19a74dacb381dda56a1d6ac6096b995403cc8fcfdb8c4493b77e74e05fd4cdR102

Reply:
Whoops looks like I missed some updates from the rest of the PR. Pulling those in now, thanks!

Reply:
@ingvagabund Updated!